src/theme/Footer/index.tsx

```diff
@@ -192,14 +198,11 @@ function Footer(): JSX.Element | null {
                   <Link href={SocialLinks.Discord}>Discord</Link>
                 </li>
                 <li>
-                  <Link href={SocialLinks.Telegram}>Forums</Link>
+                  <Link href={SocialLinks.Telegram}>Mina Research</Link>
                 </li>
                 <li>
                   <Link href={SocialLinks.GitHub}>Github</Link>
```

Comment:
oh where is the content for the social links for GitHub? We have more repositories now, let's chat (make sure to spell GitHub with the capital H)

```suggestion
                  <Link href={SocialLinks.GitHub}>GitHub</Link>
```